src/main/java/decodes/tsdb/CpCompDependsUpdater.java

```diff
@@ -1138,13 +1149,13 @@ private boolean fullEval()
 			doingFullEval = false;
 
 			// Insert all the toAdd records into the scratchpad
-			clearScratchpad();
+			clearScratchpad(dao);
 			for(CpCompDependsRecord ccd : toAdd)
 			{
 				q = "INSERT INTO CP_COMP_DEPENDS_SCRATCHPAD(TS_ID, COMPUTATION_ID)"
```

Comment:
Need to be prepared statement with bind variables. If I remember Dave's analysis correctly this was either the worst or 2nd worst offender in eating the share query space.

Databases aren't smart enough to figure out a plan can be reused without that ?, ? instead of actual data hint.